File: TypingAttributesDemo_Swift/TypingAttributesDemo_Swift/ViewController.swift
//
//  ViewController.swift
//  TypingAttributesDemo_Swift
//
//  Created by Admin on 13/05/2019.
//  Copyright © 2019 Admin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let tf = UITextView();
        assert(tf.typingAttributes != nil);
        assert(tf.typingAttributes.count == 0);
        tf.attributedText = NSAttributedString(string: "some text");
        assert(tf.typingAttributes != nil);
        assert(tf.typingAttributes.count == 1);
    }


}

File: TypingAttributesDemo_Xamarin/TypingAttributesDemo/ViewController.cs
using Foundation;
using System;
using System.Diagnostics;
using UIKit;

namespace TypingAttributesDemo
{
    public partial class ViewController : UIViewController
    {
        public ViewController(IntPtr handle) : base(handle)
        {
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
            var tf = new UITextView();
            Debug.Assert(tf.TypingAttributes != null);
            Debug.Assert(tf.TypingAttributes.Count == 0);
            tf.AttributedText = new NSAttributedString("some text");
            Debug.Assert(tf.TypingAttributes != null);
            Debug.Assert(tf.TypingAttributes.Count == 1);
        }
    }
}